GTG/gtk/browser/main_window.py

```diff
@@ -990,6 +990,26 @@ def on_add_subtask(self, widget=None):
 
             self.app.open_task(task.get_id(), new=True)
 
+    def on_add_parent(self, widget=None):
+        uid = self.get_selected_task()
```

Comment:
@zeddo123  what should do if you select multiple task? 
maybe here we can loop on selected and add those multiple task the same new parent, what do you think?

Reply:
Yeah, I agree. Having a common parent for all the selected task would be better.

Reply:
Now that I had time to think about it. Doing it this way is complicated when the selected tasks are not on the same level.

* Task A
   * Task A.1
   * Task A.2 [selected]
   * Task A.3 [selected]
* Task B
   * Task B.1 [selected]
* Task C

In this case, how should the feature behave? It's not clear where the new common parent task will be put.

Reply:
I think you could check whenever all the selected tasks have the same parent, so you could just select all Subtasks of A, which would make it more obvious what the result does. Maybe check that when you right click to disable the menu entry as needed.

Reply:
I think @Neui's idea is the best way to handle it. Code and functionality look good

Reply:
Ok, I'll work on it!